util/csvresponse.py

```diff
@@ -112,7 +114,7 @@ def _create_filter_list(self, keys):
 
     @staticmethod
     def _is_qc_param(param):
-        return QC_EXECUTED in param or QC_RESULTS in param
+        return QC_EXECUTED in param or QC_RESULTS in param or QARTOD_PRIMARY in param or QARTOD_SECONDARY in param
```

Comment:
Is param a single item or a collection? If a single item, maybe the check should be if param is in a list comprised of QC_EXECUTED, etc.  If a collection, maybe the check should make sets of both param and QC_EXECUTED, etc and check for intersection. Just a thought.

Reply:
param is the string for the name of a single parameter. The thing is, its not an exact match to the constants in question (e.g. QARTOD_PRIMARY). E.g. parameter could be "salinity_qartod_results" whereas QARTOD_PRIMARY is "qartod_results". So checking "param in QARTOD_PRIMARY" would return false whereas checking "QARTOD_PRIMARY in param" would return true. For that reason, checking if param is in a list doesn't work.

Reply:
Oh, I remember that now. Been awhile since I did any work there. Thanks for the reply.